frontend/app/src/js/main.de.js

```diff
@@ -0,0 +1,9 @@
+// @flow
+
+import { initializeLocalization } from '../../../lib/js/localization';
+import translations               from '../../../lib/localization/de.yml';
+import conqueryTranslations       from '../localization/de.yml';
+
+initializeLocalization('de', translations, conqueryTranslations);
```

Comment:
We should move the loading of `lib/localizations/[...].js` to the `initializeLocalization` to make the usage as library easier.
Even greater would be to only pass the locale string (e.g. 'de') and a function which loads localizations from app and lib and merges them.
This should be handled in another issue.